src/useTock.ts

```diff
@@ -72,6 +74,14 @@ function mapCard(card: any): Card {
   } as Card;
 }
 
+function mapImage(image: any): Image {
+  return {
+    title: image.title ? image.file.title : null,
+    imageUrl: image.file ? image.file.url : null,
```

Comment:
Style nitpick: maybe we could start using null-coalescing operators here?

Reply:
Something like that  ` image.file?.url ?? null ` @Fabilin ?

Reply:
That's what I was thinking about yes, although if `null` is not specifically required we could also omit it